ibm_catalog.json

```diff
@@ -0,0 +1,103 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-icd-redis",
+      "label": "ICD Redis Module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "redis",
+        "icd"
+      ],
+      "short_description": "Implements an instance of IBM Cloud Databases for Redis.",
+      "long_description": "Use this module to provision and configure an IBM Cloud Database for Redis instance.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-redis/main/images/redis_icon.svg",
+      "features": [
+        {
+          "title": "Create Redis instance",
+          "description": "Create IBM Cloud Database for Redis instance."
+        },
+        {
+          "title": "Attach Users",
```

Comment:
Managing Users and Roles

Reply:
just don't capitalize Users or Roles unless at beginning of sentence